public/styles/style.css

```diff
@@ -117,4 +115,14 @@ body {
       display: flex;
       flex-direction: column;
       width: 100%;
+  }
+
+  .artist-img {
+      min-width: 320px;
+      min-height: 320px;
+  }
+
+  .album-img {
+      min-width: 300px;
+      min-height: 300px;
```

Comment:
Maybe you have noticed, but your images overlap when you are in artists page (when you search artist)

Here when you set min-width or min-height, keep in mind that if the image provided is bigger than this -> it will be displayed in original size. 

For artists for example, you could set smaller dimensions, and make them fixed -> so they are all the same size.
Or put those images inside a div, and make div of fixed size-so they align nice on the page. and then format img accordingly-to fit inside the div